_episodes/03-create.md

```diff
@@ -138,9 +148,9 @@ $ nano draft.txt
 > [Vim](http://www.vim.org/) (both of which require more time to learn), 
 > or a graphical editor such as
 > [Gedit](http://projects.gnome.org/gedit/). On Windows, you may wish to
-> use [Notepad++](http://notepad-plus-plus.org/).  Windows also has a built-in
-> editor called `notepad` that can be run from the command line in the same
-> way as `nano` for the purposes of this lesson.  
+> use [Notepad++](http://notepad-plus-plus.org/) or [nano](https://github.com/lhmouse/nano-win).
```

Comment:
We are trying to give learners examples of text editors other than `nano` here.

Reply:
I'm not trying to argue here. Please note that not every leaner can make "nano" work as they or this tutorial wish (considering OS differences, etc) if you are really doing the teach. And every one wants to experience what we are doing and what's happening.  If the example not working, what is the point of example :)

Reply:
I agree, and sorry - I should have been more clear: I only mean that the purpose of this callout is expressly to give learners examples of **other** (more powerfuly/friendly) editors than `nano`:

> However, because of this trait, it may not be powerful enough or flexible enough for the work you need to do after this workshop

So mentioning `nano` as an example for Windows is unnecessary.